### DISCUSSION TITLE
表单项是按钮或者按钮组怎么用schema的方式配置

### DISCUSSION THREAD

**Discussion by wjwjwjw** [alibaba/formily#1335] [Q&A 2.x] [answered]
表单项是按钮或者按钮组怎么用schema的方式配置

**Comment by janryWang** [comment, marked as answer]
直接components传入，然后用void类型字段，指定x-component

**Reply by knight-peter** [reply]
我是用vue版本的，引入antdv的Button，用x-content显示按钮文字，按钮显示没有问题。
因为想把表单页面都用json渲染，所以得确定按钮上得绑定点击提交的事件。这个没在文档上找到对应的例子。
请问如何绑定v-on:click或者别的事件方法，在这个方法里获取整个form的数据？

**Reply by janryWang** [reply]
form实例都是你传入的，所以你直接消费form.values即可
